.github/workflows/ontology.yml

```diff
@@ -0,0 +1,45 @@
+name: Uglify Ontology
+on:
+  push:
+    paths:
+      - '**'
+      - '!docs/**'
+
+jobs:
+  uglify_ontology:
+    name: Uglify Ontology
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+
+      - name: Uglify
+        uses: docker://swipl:8.0.3
+        with:
+          args: swipl -f prolog/uglify_ease.pl -t uglify_ease
+
+      - name: Convert
+        uses: ./.github/actions/convert
+        with:
+          args: python3 ./scripts/uglify_ease_as_owl.py
```

Comment:
In fact it does not seem to make much difference anymore, maybe I had issues with caching etc.?
It could be that the fix was really to change the package:// URLs and not converting it, so essentially we could remove this step.